### PR TITLE
added new page for map docs

### LINKED ISSUE #4956: [DOCS] Add to docs how map works and is populated
### Description

Add the API and information about how the map works into the docs for LinkFree to decrease confusion about this feature.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by amandamartin-dev**
## Fixes Issue #4956 

Closes #4956 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
location of new section

content

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4986"></a>

### CODE REVIEW

File: pages/docs/map.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+In addition to the tech listed in the <Link href="https://github.com/EddieHubCommunity/LinkFree/blob/main/README.md">readme</Link>, the map utilizes open source mapping library <Link href="https://leafletjs.com/">LeafletJS</Link> and <Link href="https://nominatim.openstreetmap.org/ui/search.html?q=">Nominatum</Link> by Open Street Map for retrieving Lat/Long values.
+
+## How do I add myself to the map?
+
+The map data is based on your <Link href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#setting-your-location-and-time-zone">GitHub profile location</Link>. This data is refreshed every 7 days. If you have recently updated your GitHub profile, please wait for the data to refresh until after this period of time.
+
+## I have a location in my profile and it has been longer than 7 days but I still do not see myself.
+
+Test the <Link href="https://nominatim.openstreetmap.org/ui/search.html?q=">Nominatum API endpoint</Link> swith your exact profile location to see if it returns a value
```
Comment (eddiejaoude) [inline pages/docs/map.mdx:18]: typo 

```suggestion
Test the <Link href="https://nominatim.openstreetmap.org/ui/search.html?q=">Nominatum API endpoint</Link> with your exact profile location to see if it returns a value
```

**Review by eddiejaoude — DISMISSED** [review]
Looks great, thank you 👍  (just a small typo)

**Comment by amandamartin-dev** [comment]
> Looks great, thank you 👍 (just a small typo)

thanks! commited your edit

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**